### PR TITLE
Remove python 3.8

### PR DESCRIPTION
**Pull request by noemifrisina**

### CODE REVIEW

File: .github/workflows/code.yml [hunk 1]
```diff
@@ -35,12 +35,12 @@ jobs:
       fail-fast: false
       matrix:
         os: ["ubuntu-latest"] # can add windows-latest, macos-latest
-        python: ["3.9", "3.10", "3.11"]
+        python: ["3.10", "3.11"]
```
Comment (DominicOram) [inline .github/workflows/code.yml:38]: Should: I thought this was remove 3.8, why is this removing 3.9?

**Review by DominicOram — CHANGES_REQUESTED** [review]
Looks good, other than removing too much

File: .github/workflows/code.yml (hunk 1 above)
Comment (noemifrisina) [reply .github/workflows/code.yml:38]: It's not really, 3.9 has taken the place of 3.8 as the non-editable version for testing (a few lines down) so it shouldn't be also in the list of editable installs.

Comment (DominicOram) [reply .github/workflows/code.yml:38]: You're right, sorry

**Review by DominicOram — APPROVED** [review]

**Pull request merged**